src/misc/news/items/RecoveryCodeNews.ts

```diff
@@ -61,14 +64,12 @@ export class RecoveryCodeNews implements NewsListItem {
 
 	render(newsId: NewsId): Children {
 		const recoveryCode = this.recoveryCode
+		// toggle the load if it's not started yet
+		this.recoverCodeField.getAsync()
+
+		const RecoverCodeField = this.recoverCodeField.getSync()
```

Comment:
maybe add a comment here explaining when getSync actually returns a value that is not null.
i.e. after the dynamic import is finished and the render function is triggered again by the redraw.
but I don't have a strong opinion on this